### DISCUSSION TITLE
TypeScript + Path Aliases not Working

### DISCUSSION THREAD

**Discussion by RickEyre** [vercel/vercel#10139] [Help]
### Problem Description

I'm attempting to deploy serverless functions written in TypeScript with Vercel. I've provided a `tsconfig.json` file and have added some custom path aliases. TypeScript in my IDE works perfectly (the TypeScript configuration is working as expected), but when I deploy to Vercel the `import` statements to the path aliases fail to be included in the deployment. It looks like Vercel thinks they are node modules.

Is this a known issues or something I can fix quickly?

**Comment by emanuele-scarsella** [comment]
I'm facing the same problem so I can confirm that it hasn't been fixed yet.
At this moment I have found no practical solution for it, did you found any workaround in the meanwhile?

**Reply by RickEyre** [reply]
I just changed everything to relative paths.

**Comment by devjiwonchoi** [comment]
@RickEyre @emanuele-scarsella Just to confirm, the `serverless functions` you are talking about are the standalone express app, right?

**Reply by emanuele-scarsella** [reply]
I found the same problem with both standalone express app implemented as described [here](https://vercel.com/guides/using-express-with-vercel#standalone-express) and with the default vercel serverless functions (those described [here](https://vercel.com/docs/functions/serverless-functions/quickstart#create-a-serverless-function)).
I'm not sure it answers your question but fif you need any other information I'm at your disposal.

**Reply by devjiwonchoi** [reply]
I've opened a feature request, any suggestions or feedback are welcome. #10717
